README.md

```diff
@@ -45,7 +47,9 @@ apply plugin: 'com.fourlastor.pickle'
 pickle {
     featuresDir = 'features' // location of features inside `androidTest/src/assets`
     packageName = 'com.example.test' // package where tests will be generated
-    strictMode = false // activate/deactivate strict mode (optional, defaults to true)
+    strictMode = false // activate/deactivate strict mode (defaults to true)
+    androidTest = true // enables pickle on androidTest (defaults to true)
+    unitTest = true // enables pickle on unit tests (defaults to false)
```

Comment:
Out of scope but I have some feedback about this. 

People generally tend to copy paste what they saw. When we first implementing, we just copy pasted here and that means `strictMode` was false. We couldn't figure out what's wrong since no tests were generated. (We had missing steps). 

I think it is a good idea to keep the example to defaults even though you say `defaults to false` next.

Reply:
I'll do it now cause it's a very good point d3e33eb